main/mariadb/APKBUILD

```diff
@@ -104,7 +104,7 @@ build() {
 		-DPLUGIN_MROONGA=NO \
 		-DPLUGIN_OQGRAPH=NO \
 		-DPLUGIN_PARTITION=YES \
-		-DPLUGIN_ROCKSDB=NO \
+		-DPLUGIN_ROCKSDB=YES \
```

Comment:
does rocksdb work on all arches or only on x86_64?

Reply:
Some docs only mentioned x86_64, but my intention was to see which builds that would fail and then make corrections.